### PR TITLE
Social Media Channels - More ways to connect

### PR DESCRIPTION
**Pull request by SebastianBurke**
image

### CODE REVIEW

**Review by Garneauma — CHANGES_REQUESTED** [review]
Please add the link inside the include file instead of the working example page and add a setting in the include file like "if moreWays == true". You can add that setting to the include command in the "more ways to connect" working example.

The "gruntfile.coffee" should not be in this PR. Please revert this change.

You should always `git fetch --all` and `git reset --hard upstream/master` whenever you start working on a new PR.

Let me know if you have any question. I'll be happy to help.

**Review by Garneauma — CHANGES_REQUESTED** [review]
- Please remove all occurrences of defined parameters defined to false in the include commands.
- Add history item to log this change at line 74 of index.json-ld.
- The inline version with more ways to connect needs to be adjusted. You will need to add a class to this `<li>` like `class="more-ways"` and some custom styling in  `_base.scss`. so that it wraps under and does not have the styles of the other elements. I suggest editing line 128 to `li:not(.more-ways)`.

**Review by Garneauma — COMMENTED** [review]
One small change. Should be good after that.

File: components/gc-follow-us/_base.scss
Comment (SebastianBurke) [reply components/gc-follow-us/_base.scss:188]: This breaks the styling for the link
image

Comment (Garneauma) [reply components/gc-follow-us/_base.scss:188]: You must have put it at the wrong spot because it works for me. Make sure it is outside the "li:not(.more-ways)" definition, but inside the "&.list-inline" definition.

**Comment by Garneauma** [comment]
Pre-approved upon successful review. Waiting for formal DTO approval.

**Comment by Garneauma** [comment]
@SebastianBurke, DTO just came back and would like for the link to be underlined. Please more the ".more ways" definition to line 105 right under "&:last-child". Please also make sure it reads "&.more-ways". Also, add "text-decoration: underline" to the "a" inside ".more-ways". Thank you.

File: components/gc-follow-us/index.json-ld
Comment (Garneauma) [inline components/gc-follow-us/index.json-ld:79]:
```suggestion
			"history": [
				{
			    	"en": "July 2024 - Adding 'More ways to connect' optional link.",
					"fr": "Juillet 2024 - Ajout du lien optionnel « D'autres moyens de rester en contact ».",
				},
				{
					"en": "February 2024 - Initial implementation of the variation.",
					"fr": "Février 2024 - Implémentation initiale de la variante."
				}
```

File: components/gc-follow-us/index.json-ld
Comment (Garneauma) [inline components/gc-follow-us/index.json-ld:3]: This is the DCTerms version. Please change to 1.1. Thanks!
```suggestion
		"@version": 1.1,
```

**Review by Garneauma — APPROVED** [review]

**Pull request merged**
